README.md

```diff
@@ -28,7 +28,7 @@ There is currently no API for the OTW-Archive software. While it is something we
 
 License and Acknowledgments
 ----------
-The Archive code is licensed under [GPL](https://www.gnu.org/licenses/gpl-2.0.html) by the [Organization for Transformative Works](https://www.transformativeworks.org/).
+The Archive code is licensed under [GPL-2.0.-or-later](https://www.gnu.org/licenses/gpl-2.0.html) by the [Organization for Transformative Works](https://www.transformativeworks.org/).
```

Comment:
```suggestion
The Archive code is licensed under [GPL-2.0-or-later](https://www.gnu.org/licenses/gpl-2.0.html) by the [Organization for Transformative Works](https://www.transformativeworks.org/).
```

It looks like there’s an extra dot that should be removed, this is also in some of the other files in this PR. See https://spdx.org/licenses/GPL-2.0-or-later.html.

Reply:
Thanks for catching that, @printfn! I see now I made that typo in one spot in the issue itself, so mea culpa and my apologies, @denerose

Reply:
Thanks @printfn! A leading .- for a subordinate clause extension is actually a legalistic affectation you sometimes see in governance/legal documents so I didn't recognise it as a typo either (despite my English degree). All fixed up and consistent now.